util/config/config.go

```diff
@@ -224,7 +225,7 @@ var (
 		PasswordFile:                 "",
 		StatePruningMode:             "lowmem",
 		RecentStateCount:             1024,
-		MinPruningCompactHeights:     4096,
+		MinPruningCompactHeights:     32768,
```

Comment:
Should we test it on $5 machine which have 30k uncompact block to see if it have enough space?

Reply:
This parameter will have almost no effect on the disk usage because compact only happens when nknd starts. Typically nknd start very infrequently, e.g. once per month